resources.whatwg.org/build/deploy.sh

```diff
@@ -78,20 +77,17 @@ run_post_build_step() {
     fi
 }
 
-if [[ $BRANCH != "master" ]] ; then
-    # Branch snapshot, if not master
-    BRANCH_DIR="$WEB_ROOT/$BRANCHES_DIR/$BRANCH"
-    mkdir -p "$BRANCH_DIR"
-    curl https://api.csswg.org/bikeshed/ -f -F file=@"$INPUT_FILE" -F md-status=LS-BRANCH \
-         -F md-warning="Branch $BRANCH $BRANCH_URL_BASE$BRANCH replaced by $LS_URL" \
-         -F md-title="$TITLE (Branch Snapshot $BRANCH)" \
-         -F md-Text-Macro="SNAPSHOT-LINK $BACK_TO_LS_LINK" \
-         > "$BRANCH_DIR/index.html";
-    copy_extra_files "$BRANCH_DIR"
-    run_post_build_step "$BRANCH_DIR"
-    echo "Branch snapshot output to $BRANCH_DIR"
-else
-    # Commit snapshot, if master
+# Living standard
```

Comment:
I guess one notable change here is that local deploys will always be as-if they are master/living standard, even if they're on a branch. That's probably for the better.

Reply:
Yep. Writing much more detail in the separate PR.